diagnostics/model_test/verify_model.py

```diff
@@ -0,0 +1,185 @@
+        graph_def = tf.GraphDef()
+        graph_def.ParseFromString(f.read())
+
+    if select_device == 'CPU':
+        ngraph.disable()
+    else:
+        # run on NGRAPH
+        if not ngraph.is_enabled():
+            ngraph.enable()
+
+    with tf.Graph().as_default() as graph:
+        tf.import_graph_def(graph_def)
+
+    # Create the tensor to its corresponding example map
+    tensor_to_example_map = {}
+    for item in input_example:
+        t = graph.get_tensor_by_name(item)
+        tensor_to_example_map[t] = input_example[item]
+
+    #input_placeholder = graph.get_tensor_by_name(input_tensor_name)
+    output_tensor = graph.get_tensor_by_name(output_tensor_name)
+
+    config = tf.ConfigProto(
+        allow_soft_placement=True,
+        # log_device_placement=True,
+        inter_op_parallelism_threads=1)
+
+    with tf.Session(graph=graph, config=config) as sess:
+        output_tensor = sess.run(output_tensor, feed_dict=tensor_to_example_map)
+        return output_tensor
+
+
+def calculate_norm(ngraph_output, tf_output, desired_norm):
+    """Calculate desired_norm between vectors.
+
+    Calculate the L1/L2/inf norm between the NGRAPH and tensorflow output vectors.
+
+    Args:
+        ngraph_output: The output vector generated from NGRAPH graph.
+        tf_output: The output vector generated from tensorflow graph.
+        desired_norm: L1/L2/inf norm. 
+
+    Returns:
+        Calculated norm between the vectors.
+
+    Raises:
+        Exception: If the dimension of the two vectors mismatch.
+    """
+    if (ngraph_output.shape != tf_output.shape):
+        raise Exception('ngraph output and tf output dimension mismatch')
+
+    ngraph_output_squeezed = np.squeeze(ngraph_output)
+    tf_output_squeezed = np.squeeze(tf_output)
+
+    ngraph_output_flatten = ngraph_output_squeezed.flatten()
+    tf_output_flatten = tf_output_squeezed.flatten()
+
+    factor = np.prod(ngraph_output_squeezed.shape)
+
+    if desired_norm not in [1, 2, np.inf]:
+        raise Exception('Only L2, L2, and inf norms are supported')
+
+    return np.linalg.norm((ngraph_output_flatten - tf_output_flatten),
+                          desired_norm)
+
+
+def parse_json():
+    """
+        Parse the user input json file.
+
+        Returns:
+            A dictionary contains all the parsed parameters.
+    """
+
+    param_dict = {}
+
+    with open(os.path.abspath(args.json_file)) as f:
+        parsed_json = json.load(f)
+        frozen_graph_location = parsed_json['frozen_graph_location']
+        input_tensor_name = parsed_json['input_tensor_name']
+        output_tensor_name = parsed_json['output_tensor_name']
+        l1_norm_threshold = parsed_json['l1_norm_threshold']
+        l2_norm_threshold = parsed_json['l2_norm_threshold']
+        inf_norm_threshold = parsed_json['inf_norm_threshold']
+        input_dimension = parsed_json['input_dimension']
+        batch_size = parsed_json['batch_size']
+
```

Comment:
I don't think you need the variables above, you can directly parse the json and assign to the dictionary.